CHANGELOG.md

```diff
@@ -13,6 +13,7 @@
 * [BUGFIX] querier: Don't cache context.Canceled errors for bucket index. #7620
 * [BUGFIX] Store-gateway: account for `"other"` time in LabelValues and LabelNames requests. #7622
 * [BUGFIX] Query-frontend: Don't panic when using the `-query-frontend.downstream-url` flag. #7651
+* [BUGFIX] Query-frontend: close the response body upon completion of the request. #7654
```

Comment:
[nit] This likely won't make much sense to an end-user. Perhaps something like "fix memory leak on every request" would be more appropriate?

Reply:
Absolutely, 5d8e8bo